instrument/IN4_Parameters.xml

```diff
@@ -17,7 +17,34 @@
 			See http://muparser.sourceforge.net/mup_features.html#idDef2 for available 
 			operators -->
 		<parameter name="formula_eff" type="string">
-			<value val="0.951*exp(-0.0887/sqrt(e))*(1.0-exp(-5.597/sqrt(e)))" />
+			<value val="0.951*exp(-0.0887/sqrt(e))*(1-exp(-5.597/sqrt(e)))" />
```

Comment:
These changes will go out to users immediately once merged, not with the next release.  The look safe to me, but you might want to double check.

Reply:
I think the users should be happy as the old efficiency corrections for IN5 and IN6 were incorrect.

For IN4, the formulas need a bit more work, as described in #20459.